src/reports/summary.cpp

```diff
@@ -426,28 +428,29 @@ wxString mmReportSummaryByDate::getHTMLText()
         totBalanceData.push_back(dateStart.FormatISODate());
         for (j=0; j<5; j++)
             balancePerDay[j] = 0.0;
-        for (j=0; j<4; j++)
+        for (j=0; j<5; j++)
         {
             i = 0;
             for (const auto& account: Model_Account::instance().all())
             {
-                if ((j == 0 && Model_Account::type(account) == Model_Account::CHECKING) ||
-                    (j == 1 && Model_Account::type(account) == Model_Account::CREDIT_CARD) ||
-                    (j == 2 && Model_Account::type(account) == Model_Account::TERM) ||
-                    (j == 3 && Model_Account::type(account) == Model_Account::INVESTMENT))
+                if ((j == 0 && Model_Account::type(account) == Model_Account::CASH) ||
```

Comment:
I don't understand this code.
~~~~~
for (const auto& account: Model_Account::instance().all())
    balancePerDay[account.ACCOUNTTYPE] += arBalance[i];
~~~~~